src/k8s/package.dhall

```diff
@@ -4,4 +4,6 @@ let Kubernetes = ../deps/k8s/schemas.dhall
 
 let Generate = ./generate.dhall
 
-in  { Configuration, Kubernetes, Generate }
+let Util = { EnvToK8s = ../util/functions/environment-to-k8s.dhall }
```

Comment:
Note I added your utility function as a package export (so that users can reference it from the same variable at the top of their pipeline file)

Reply:
nice